fulcio-spec.md

```diff
@@ -24,7 +24,7 @@ To issue code signing certificates, Sigstore uses Fulcio, a certificate authorit
 
 ### 2.1. Timestamping
 
-The certificates that Fulcio issues are short-lived. By using ephemeral keys and short-lived certificates, Fulcio avoids the need for revocation lists. To verify a short-lived certificate, a timestamping service is needed to verify that the issued certificate was valid during artifact signing. Sigstore clients can leverage either Sigstore's own timestamping service ([Spec: Timestamping Service](https://docs.google.com/document/d/1FoRHXejIhXwEai0RS3iRsN1HfCV16fJOp582Vl8KA7A/edit#heading=h.dal05eqvj5ql)) that implements [RFC 3161](https://www.ietf.org/rfc/rfc3161.txt) to provide a trusted timestamp.
+The certificates that Fulcio issues are short-lived. By using ephemeral keys and short-lived certificates, Fulcio avoids the need for revocation lists. To verify a short-lived certificate, a timestamping service is needed to verify that the issued certificate was valid during artifact signing. Sigstore clients can leverage Sigstore's own timestamping service that implements [RFC 3161](https://www.ietf.org/rfc/rfc3161.txt) to provide a trusted timestamp.
```

Comment:
maybe we could reflect the fact that Fulcio is not an obligatory component in the signing flow ("private infrastructure") - for example, say `The certificates that Fulcio (or equivalent CA) issues are short-lived.`

Reply:
This spec is specific to describing Fulcio, a certificate profile for code signing with identities so the 'other CA' is implicit. There are other docs where we can emphasize the non-obligatory nature of Fulcio (e.g. public deployment spec).